lib/whois/record/parser/whois.corporatedomains.com.rb

```diff
@@ -23,8 +23,29 @@ class WhoisCorporatedomainsCom < BaseIcannCompliant
         self.scanner = Scanners::BaseIcannCompliant, {
             pattern_available: /^No match for/
         }
-      end
 
+        property_supported :domain_id do
+          node('Domain ID')
+        end
+
+        property_supported :expires_on do
+          node('Registry Expiry Date') do |value|
+            parse_time(value)
+          end
+        end
+
+        property_supported :registrar do
+          return unless node('Sponsoring Registrar')
```

Comment:
This is incorrect. The CorporateDomain response doesn't include this data. You are taking it from the Verisign response.